packages/common/src/hdwallets.ts

```diff
@@ -71,11 +71,11 @@ export const bigNumberFieldsToStrings = (
     (transaction, txnProperty) => ({
       ...transaction,
       ...((transaction[txnProperty as keyof TransactionRequest] as BigNumber)
-        .toHexString
+        .toString(16)
```

Comment:
It looks like this first expression is not meant to call the `toString` method and it is actually just checking if it is a big number or not. You might be able to use an `instanceof` check instead to see if the value is a Bignumber or not.